From 0389d15a0f13f20d222286cd0613dad566e3fad3 Mon Sep 17 00:00:00 2001
From: Michael Shamis <michaelsh@marvell.com>
Date: Wed, 13 Jan 2021 23:03:20 +0200
Subject: [PATCH 1153/1921] arm64: defconfig: enable CONFIG_CRYPTO_DEV_SAFEXCEL

Enable CONFIG_CRYPTO_DEV_SAFEXCEL in
marvell_v8_sdk_defconfig

Signed-off-by: Michael Shamis <michaelsh@marvell.com>
Change-Id: Idee92644052f7c5ae7610a442f9842460e1529c3
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/43908
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index 17c10b485810..26492b7d85d9 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -836,6 +836,7 @@ CONFIG_NLS_ISO8859_1=y
 CONFIG_SECURITY=y
 CONFIG_CRYPTO_ECHAINIV=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
+CONFIG_CRYPTO_DEV_SAFEXCEL=m
 CONFIG_CMA_SIZE_MBYTES=256
 CONFIG_PRINTK_TIME=y
 CONFIG_DEBUG_INFO=y
-- 
2.31.1

